plugins/clipboard-manager/guest-js/index.ts

```diff
@@ -11,6 +11,7 @@
 import { invoke } from "@tauri-apps/api/core";
 
 type ClipResponse = Record<"plainText", { text: string }>;
+type ClipHtmlResponse = Record<"html", { html: string, alt_html: string }>;
```

Comment:
The Rust enum has `rename_all = "camelCase"`
```suggestion
type ClipHtmlResponse = Record<"html", { html: string, altHtml: string }>;
```